homeassistant/components/volkszaehler/sensor.py

```diff
@@ -44,6 +51,9 @@
         vol.Optional(CONF_HOST, default=DEFAULT_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_FROM, default=DEFAULT_FROM): cv.string,
+        vol.Optional(CONF_TO, default=DEFAULT_TO): cv.string,
+        vol.Optional(CONF_TLS, default=DEFAULT_TLS): cv.boolean,
```

Comment:
Wait, this actually adds changes to the YAML configuration now, however, we no longer accept YAML configuration for integrations that connect to devices or services.

Instead, a configuration flow via the UI should be used.

For more information about this decision see:

- ADR-0007: <https://github.com/home-assistant/architecture/blob/master/adr/0007-integration-config-yaml-structure.md#decision>
- ADR-0010: <https://github.com/home-assistant/architecture/blob/master/adr/0010-integration-configuration.md#decision>

To get started on the integration configuration, see our developer documentation:

<https://developers.home-assistant.io/docs/config_entries_config_flow_handler>